demo/index.html

```diff
@@ -28,6 +28,14 @@
     The <code>value</code> stored at <code>key="info"</code> is <code><meta-test></meta-test></code>.
   </div>
 
+  <div class="vertical-section vertical-section-container centered">
```

Comment:
This is an existing issue, but here and in the above card, consider removing the `vertical-section-container` class.

As is,
the text is
very
vertical
and hard
to read.

If a bit poetic.